server/api/products.js

```diff
@@ -0,0 +1,78 @@
+const router = require("express").Router();
+
+// const { Product } = require("../db/index.js");
+
+
+router.get('/', (req, res, next) => {
+	Product.findAndCountAll()
+	.then(products => res.send(products))
+	.catch(e => {
+		res.status(404)
+		next(e);
+	})
+})
+
+router.get('/:id', (req, res, next) => {
+	Product.findByPk(req.params.id)
+	.then(product => res.send(product))
+	.catch(e => {
+		res.status(404);
+		next(e);
+	})
+})
+
+//Creates a new product.
+//Sets falsy field in req.body.productDescription to be null.
+//Sets falsy field in req.body.inventory to 0.
+router.post('/', (req, res, next) => {
+	const {
+		productName,
+		productDescription,
+		unitPrice,
+		inventory,
+	} = req.body
+
+	Product.create({
+		productName,
+		productDescription: productDescription || null,
```

Comment:
Cool. Just noting that client needs to check if description is null b4 doing any string manipulation.